client/src/app/site/interaction/components/interaction-container/interaction-container.component.scss

```diff
@@ -44,7 +44,7 @@ $radius: 5px;
 .video-player {
     display: block;
     width: 500px;
-    height: 280px;
+    max-height: 280px;
```

Comment:
@emanuelschuetze this was never supposed to be `max-height`. Consult be for changes before breaking stuff.